config/dev.js

```diff
@@ -46,4 +46,8 @@ module.exports = {
   extensionWorkshopUrl: 'https://extensionworkshop-dev.allizom.org',
 
   enableFeatureLinkToNewBlog: true,
+
+  experiments: {
+    '20210531_download_funnel_experiment': true,
```

Comment:
are these random numbers or do they mean something?  (if it's May 31st, why May 31st?)

Reply:
It does stand for May 31st, and that's because that's the day I wrote that code. The date doesn't particularly matter - it's just a way to try to make it easier to ensure that each `id` is unique.